lib/tasks/create_missing_whitehall_assets.rake

```diff
@@ -0,0 +1,53 @@
+# rubocop:disable Metrics/BlockLength
+task create_missing_whitehall_assets: :environment do
+  placeholder_path = Rails.root.join('tmp/whitehall-attachment-placeholder.txt')
+  File.open(placeholder_path, 'w') do |file|
+    file.puts('whitehall-attachment-placeholder.txt')
```

Comment:
Minor: It might be nice to add some more useful explanatory text in there, e.g. reference this PR, explain that the attachment has been replaced, etc.

Reply:
Good idea! We've updated the text in the commit titled "Improve content of temporary file use in Rake task".